frontend/www/js/omegaup/contest/new.ts

```diff
@@ -19,20 +19,26 @@ OmegaUp.on('ready', () => {
           update: false,
           initialStartTime: startTime,
           initialFinishTime: finishTime,
+          invalidParameterName: this.invalidParameterName,
         },
         on: {
           'create-contest': (contest: omegaup.Contest): void => {
             api.Contest.create(contest)
               .then((data) => {
+                this.invalidParameterName = '';
                 window.location.replace(
                   `/contest/${contest.alias}/edit/#problems`,
                 );
               })
-              .catch(ui.apiError);
+              .catch((error) => {
+                ui.apiError(error);
+                this.invalidParameterName = error.parameter || '';
+              });
           },
         },
       });
     },
+    data: { invalidParameterName: '' },
```

Comment:
```suggestion
    data: { invalidParameterName: <null|string>null },
```

aquí hay que usar esto para que TypeScript no se espante en L28 y L35 porque se está usando un tipo de datos distinto.